tutorials/single_phase_flow.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Please fix "In this tutorial will solve ". </p><p>Please include (and optionally improve) the following addition if you find it useful. "In PorePy, all class types except solution strategies are treated as Mixins. The solution strategy also defines an implicit contract between the functions used to run a simulation (run_time_dependent_problem) and the rest of the model."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/878/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>A slight modification: In PorePy all classes relating to models except solution strategies ...'</p>